libnavigation-ui/src/main/res/values/colors.xml

```diff
@@ -62,7 +62,7 @@
   <color name="md_grey_800">#424242</color>
   <color name="md_grey_900">#212121</color>
 
-  <!-- Puck Colors -->
+  <!-- Default Puck -->
   <color name="jungleAccentOrange">#FF9900</color>
```

Comment:
Could we remove `jungle` from `jungleAccentOrange`? Also for consistency, could we use snake case and similar convention as the rest of the colors?

Reply:
This was a copy/paste from the driver app. I didn't have any reason to think the name of the color should be changed. I can change it here though.